src/mmrelay/constants/migration.py

```
@@ -0,0 +1,37 @@
+
+__all__ = [
+    "MIGRATION_MAX_RETRIES",
+    "MIGRATION_INITIAL_RETRY_DELAY",
+    "MIGRATION_MAX_RETRY_DELAY",
+    "MIGRATION_STAGING_DIRNAME",
+    "MIGRATION_BACKUP_DIRNAME",
+    "MIGRATION_LOCK_FILENAME",
+    "TEMP_DEVICE_NAME_LOGOUT",
+    "MIGRATION_MIN_FREE_SPACE_BYTES",
+    "DEFAULT_SERVICE_ARGS_SUFFIX",
+]
```

Ruff (0.15.7)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)